tests/data/beta_data.py

```diff
@@ -72,3 +72,7 @@ def exp_test_data(self):
     def christoffels_shape_test_data(self):
         smoke_data = [dict(n_samples=10)]
         return self.generate_tests(smoke_data)
+
+    def sectional_curvature_test_data(self):
+        smoke_data = [dict(n_samples=10, atol=1e-8)]
```

Comment:
should we hardcode `atol` or use the new approach? (see e.g. `dirichlet_data`)

Reply:
Good catch, I've changed.